src/AzSK.ADO/Framework/Listeners/LogAnalytics/LogAnalyticsOutput.ps1

```diff
@@ -214,13 +214,15 @@ class LogAnalyticsOutput: ListenerBase
 					#publish to primary workspace
 					if(-not [string]::IsNullOrWhiteSpace($settings.LAWSId) -and [LogAnalyticsHelper]::IsLAWSSettingValid -ne -1)
 					{
-						[LogAnalyticsHelper]::PostLAWSData($settings.LAWSId, $settings.LAWSSharedKey, $lawBodyByteArray, $settings.LAType, 'LAWS')
+						#Out-null is added as this function return true/false based data is posted or not in LA, here not doing any operation on returned result.
```

Comment:
#Out-null is added as this function will return boolean  based on whether data is posted to LA or not, but  here we are not doing any operation on return value.